alcs-api/src/providers/typeorm/migrations/1665175956310-rename_decision_to_review.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class renameDecisionToReview1665175956310 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    queryRunner.query(`
+        UPDATE public."card_status" SET "label" ='Ready for Review Sent / Going to Next Review Meeting' WHERE "uuid" = '64944bb8-f2f2-4709-9062-214f5c4d3187'
```

Comment:
This can cause issues with displaying column headers. Words may be split strangely in column header.

Reply:
Good idea, updated with some nbsp; to break where we want.

Reply:
The only catch maybe with the workflow dropdown if that one is using "text" for rendering